gips/0081-indexing-payments.md

```diff
@@ -0,0 +1,165 @@
+---
+GIP: 0081
+Title: A New Proposal for Indexing Payments
+Authors: Matias <matias@edgeandnode.com>, Pablo <pablo@edgeandnode.com>
```

Comment:
For unambiguous simple front-matter parsing I suggest quoting at least when a list if supplied, although perhaps less important for a single member.
```suggestion
Authors: "Matias <matias@edgeandnode.com>", "Pablo <pablo@edgeandnode.com>"
```